### ISSUE TITLE
Some AutoIt Binary just have empty return.

### ISSUE THREAD

**Issue by JL1829** [x0r19x91/UnAutoIt#1]
Hi: 

The UnAutoIt tool is cool, it do help my research and study a lot, really appreciate it. 

recently I found there's some AutoIt Binary that UnAutoIt just return empty `stdout` and empty `stderr`, but the `returncode` is `0`.

The binary hash is: `0D3987D155A6259422A13E11789E231EDD3653E12A591E00981804FDF8DF959A`

When I `list` this AutoIt binary, it return like this: 

```shell
UnAutoIt list Downloads/bin-2/0D3987D155A6259422A13E11789E231EDD3653E12A591E00981804FDF8DF959A
+----+------+------+------+------+
| ID | NAME | PATH | SIZE | TYPE |
+----+------+------+------+------+
+----+------+------+------+------+

In [14]: command = [
    ...:     bin_path,
    ...:     'extract-all',
    ...:     '--output-dir',
    ...:     out_path,
    ...:     source_path]

In [15]: stdout = subprocess.run(command, timeout=90, capture_output=True)

In [23]: stdout
Out[23]: CompletedProcess(args=['PycharmProjects/autoit/UnAutoIt/UnAutoIt', 'extract-all', '--output-dir', 'Downloads/bin-2/output', 'Downloads/bin-2/0D3987D155A6259422A13E11789E231EDD3653E12A591E00981804FDF8DF959A'], returncode=0, stdout=b'', stderr=b'')
```

Is this a limitation? any way to debug it?

**Comment by x0r19x91** [comment]
Thanks for the review
You can debug from source using vscode. Or you can send me the binary, I will check if there's any bug in the tool that gets triggered by the binary

**Comment by JL1829** [comment]
[mli_sample.zip](https://github.com/x0r19x91/UnAutoIt/files/6364338/mli_sample.zip)
Hi, 
Thanks for the reply. 

Here is the binary, it's malicious.

**Comment by JL1829** [comment]
@x0r19x91 Hi: 

Do you have any finding?
recently I have parsed some samples that have the same problem, if you need it, I can send it over here.

Thanks
//Johnny

**Comment by x0r19x91** [comment]
It will be great if you can send the samples. I'm working on a fix for this issue (redesigning it entirely)

**Comment by x0r19x91** [comment]
The issue has been fixed. Please checkout the latest release

**Comment by JL1829** [comment]
Thanks, I will check, do you still need some more samples?

**Comment by x0r19x91** [comment]
Yes sure, I can look for if there are any other bugs

**Comment by JL1829** [comment]
Hi I have tried, still lot's of empty extraction. 

here I have packed around 160 samples: [download here](https://drive.google.com/file/d/1YCrE7gx5Jcki9pPO2-w89oL9oUgtFypg/view?usp=sharing)

some of them are malicious, please work in isolated env

**Comment by x0r19x91** [comment]
Did you use `extract-all` command. If yes, could you try using `list` command to get the id's of the artifacts and then use the `extract` command with id

**Comment by JL1829** [comment]
Hi, actually I just use `list` command to see the new fix works or not, I see it return an empty table, then I didn't process to `extract-all`

**Comment by x0r19x91** [comment]
> Hi I have tried, still lot's of empty extraction.
> 
> here I have packed around 160 samples: [download here](https://drive.google.com/file/d/1YCrE7gx5Jcki9pPO2-w89oL9oUgtFypg/view?usp=sharing)
> 
> some of them are malicious, please work in isolated env

It seems the link is not usable. Can you please use mega.nz

**Comment by JL1829** [comment]
Try this mega.nz [link](https://mega.nz/file/fbhEEJCB#6PAWS86eQDaTGMKsWLAMtcIGlrqK3_0nw1Uqcs6ZVsM)

**Comment by JL1829** [comment]
Hi Bro: 

The link accessible? Any finding of the samples?

**Comment by x0r19x91** [comment]
yes, Thanks. There are many samples. and some of them are non standard
